__tests__/run.test.ts

```diff
@@ -0,0 +1,63 @@
+import { fromLiteralsToFromFile } from "../src/run"
```

Comment:
is `run.test.ts` a convention? Or can you just call it `L0.ts`?

Reply:
*.test.ts represents the test case for a particular file in this case it is 'run'. This would make more sense than having 'L0' and having all tests in it.